File: Exercism Codes/JavaScript/vehicle-purchase.js
export function needsLicense(kind) {
  return kind === 'car' || kind === 'truck';
}
export function chooseVehicle(option1, option2) {
  let options = [option1, option2];
  options.sort();
  return options[0] + " is clearly the better choice.";
}
export function calculateResellPrice(originalPrice, age) {
  if(age < 3) {
    originalPrice *= 0.80;
  }
  if(age >= 3 && age <= 10) {
    originalPrice *= 0.70;
  }
  if(age > 10) {
    originalPrice *= 0.50;
  }
  return originalPrice;
}

File: Exercism Codes/C++/Vehicle Purchase/vehicle_purchase.cpp
#include "vehicle_purchase.h"

namespace vehicle_purchase {

    bool needs_license(std::string kind) {
        return kind == "car" || kind == "truck";
    }

    std::string choose_vehicle(std::string option1, std::string option2) {
        return ((option1 < option2) ? option1 : option2) + " is clearly the better choice.";
    }

    double calculate_resell_price(double original_price, double age) {
        if (age < 3) return original_price * 0.8;
        else if (age < 10) return original_price * 0.7;
        else return original_price * 0.5;
    }
}